LayoutTests/imported/w3c/web-platform-tests/interfaces/webrtc-encoded-transform.idl

```diff
@@ -48,40 +64,42 @@ enum RTCEncodedVideoFrameType {
 };
 
 dictionary RTCEncodedVideoFrameMetadata {
-    long long frameId;
-    sequence<long long> dependencies;
+    unsigned long long frameId;
+    sequence<unsigned long long> dependencies;
     unsigned short width;
     unsigned short height;
-    long spatialIndex;
-    long temporalIndex;
-    long synchronizationSource;
-    sequence<long> contributingSources;
+    unsigned long spatialIndex;
+    unsigned long temporalIndex;
+    unsigned long synchronizationSource;
+    octet payloadType;
+    sequence<unsigned long> contributingSources;
+    long long timestamp;    // microseconds
 };
 
 // New interfaces to define encoded video and audio frames. Will eventually
 // re-use or extend the equivalent defined in WebCodecs.
 [Exposed=(Window,DedicatedWorker)]
 interface RTCEncodedVideoFrame {
     readonly attribute RTCEncodedVideoFrameType type;
-    readonly attribute unsigned long long timestamp;
+    readonly attribute unsigned long timestamp;
```

Comment:
this isn't in https://w3c.github.io/webrtc-encoded-transform/#RTCEncodedVideoFrame-interface